models/meeting.js

```diff
@@ -4,6 +4,21 @@ import { query, sparqlEscapeString, sparqlEscapeUri } from "mu";
 import validateMeeting from "../support/validate-meeting";
 const dateFormat = process.env.DATE_FORMAT || 'dd/MM/yyyy HH:mm';
 
+const articlesBasedOnClassifcationMap = {
+  'http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/4955bd72cd0e4eb895fdbfab08da0284': 'de',
```

Comment:
perhaps add a comment on each uri so we have an idea which one it is, or use named constants

Reply:
I'm thinking about the named constants,  I think I will need them with the other issue of the voting, I will do that one, and then decide on what it's the cleanest way to do it